scipy/stats/_morestats.py

```diff
@@ -11,8 +11,14 @@
 from scipy import optimize, special, interpolate, stats
 from scipy._lib._bunch import _make_tuple_bunch
 from scipy._lib._util import _rename_parameter, _contains_nan, _get_nan
-from scipy._lib._array_api import (array_namespace, xp_minimum, size as xp_size,
-                                   xp_moveaxis_to_end)
+
+from scipy._lib._array_api import (
+    array_namespace,
+    size as xp_size,
+    xp_minimum,
+    xp_moveaxis_to_end,
+    xp_vector_norm,
+)
```

Comment:
This is OK for here, if you'd like, but I'd prefer that we not begin to convert imports like this individually. If SciPy adopts Black, we can do this all at once.